src/util/pg.py

```diff
@@ -1015,7 +1029,7 @@ def from_unquoted(cls, cr, list_):
         quoted = [quote_ident(c, cr._cnx) for c in list_]
         return cls(list_, quoted)
 
-    def using(self, leading_comma=False, trailing_comma=False, alias=None):
+    def using(self, leading_comma=KEEP_CURRENT, trailing_comma=KEEP_CURRENT, alias=KEEP_CURRENT):
```

Comment:
I think we could use "standard" (built-in) defaults. An empty alias cannot be confused with a valid alias ever.
```suggestion
    def using(self, leading_comma=None, trailing_comma=None, alias=""):
```

Reply:
I thought of this alternative, but I find `None` not really explicative of the behavior.